src/uwtools/resources/FV3Forecast.jsonschema

```diff
@@ -0,0 +1,106 @@
+{
+  "title": "FV3 Forecast config",
+  "description": "This document is to validate forecast config files from SRW, HAFS, Global",
+  "type": "object",
+  "properties": {
+    "platform": {
+      "description": "attributes of the platform",
+      "type": "object",
+      "properties": {
```

Comment:
I think probably here we only want to start with the settings that we require in the workflow-tools driver. In the tests we currently have, we expect `static` and `cycledep` sections. 

Field table updates may also be handled here, perhaps we'd want to enforce something like a `field_table` section, and then we can't say too much about which sections are there, but each of those sections require entries like this:
```
  longname: specific humidity
  units: kg/kg
  profile_type: 
    name: fixed
    surface_value: 2.0
```

We might also be interested in `run_cmd`, `exec_name`, and optional kwargs for the run command.

Many of the settings included below are for SRW, specifically and may or may not relate directly to the forecast task. I think it makes sense to build out this schema as we build out the driver to make sure we have only the information we need.

Reply:
Oh good point, thanks! I was kind of going off of the comments but still left a few things in extra. I'll scrub some of that out and add a few